primitives/consensus/common/src/import_queue.rs

```diff
@@ -43,6 +43,8 @@ pub use basic_queue::BasicQueue;
 mod basic_queue;
 pub mod buffered_link;
 
+pub type DefaultQueue<Block> = BasicQueue<Block, sp_trie::PrefixedMemoryDB<BlakeTwo256>>;
```

Comment:
Where is this used?

Reply:
I'm not sure it is in this PR, but it definitely helps simplify the import queue types.

Reply:
Please remove if it is not used. If you need it for another pr, add it in the pr where it is required. Otherwise this will may stay here and will never be used.